pyvista/core/_vtk_core.py

```diff
@@ -432,6 +432,7 @@
 from vtkmodules.vtkIOXML import vtkXMLWriter
 
 with contextlib.suppress(ImportError):
+    from vtkmodules.vtkImagingMorphological import vtkImageConnectivityFilter
```

Comment:
Suppression is only needed if the import fails for older supported vtk versions (e.g. fails CI testing), but this filter was introduced before VTK 9.
```suggestion
from vtkmodules.vtkImagingMorphological import vtkImageConnectivityFilter

with contextlib.suppress(ImportError):
```